### PR TITLE
sceNp: Fix ExitSpawn, Implement VerifyUpgradeLicense

### PR DESCRIPTION
**Pull request by AniLeo**
Fixes sceNpDrmProcessExitSpawn and sceNpDrmProcessExitSpawn2 functions

The problem was that first argument klicensee was missing, therefore shifting every other argument out of place and throwing an access violation at the end.

Before:
path: !AjɨΰｘƵ6
argv: 0xd000fc48
envp: 0xabc000
data: 0x0
data_size: 0x0
prio: 0
flags: 1001

After:
klicensee: 0x18978
path: /dev_hdd0/game/NPUB31430/USRDIR/reboot.self
argv: 0x278000
envp: 0x0
data: 0x0
data_size: 0x0
prio: 1001
flags: 4611686018427387936

Tested with Child of Light Trial [NPUB31430]

### CODE REVIEW

**Comment by Nekotekina** [comment]
sys_game doesn't have additional arg

**Comment by AniLeo** [comment]
@Nekotekina I have fixed it now

**Comment by Nekotekina** [comment]
It's possible that by injecting npDrmIsAvailable call in these functions, the executables will be decrypted.

**Comment by SakataGintokiYT** [comment]
@AniLeo 
Can you implement this function ? it is required for Plants vs. Zombies and Zuma
sceNpDrmVerifyUpgradeLicense / sceNpDrmVerifyUpgradeLicense2

**Comment by AniLeo** [comment]
@SakataGintokiYT done, but Plants Vs Zombies now crashes on something else.

File: rpcs3/Emu/Cell/Modules/sceNp.cpp [hunk 1]
```diff
@@ -51,9 +51,9 @@ s32 npDrmIsAvailable(u32 k_licensee_addr, vm::cptr<char> drm_path)
 
 	if (k_licensee_addr)
 	{
-		for (s32 i = 0; i < 0x10; i++)
+		for (s8 i = 0; i < 0x10; i++)
 		{
-			k_licensee[i] = vm::read8(k_licensee_addr + i);
+			k_licensee[i] = *k_licensee_addr + i;
```
Comment (danilaml) [inline rpcs3/Emu/Cell/Modules/sceNp.cpp]: Wouldn't it behave differently, from `vm::read8(k_licensee_addr + i);`?
Shouldn't it be `*(k_licensee_addr + i);`?

Comment (AniLeo) [inline rpcs3/Emu/Cell/Modules/sceNp.cpp]: Yeah it does, sorry about that. Will commit

**Pull request merged**